### ISSUE TITLE
Add moving average view

### ISSUE THREAD

**Issue by smarter** [liufengyun/bench-web#3]
Because of the high variance, It can be a bit hard to read the benchmark graphs. One way to make it easier to pick up trends would be to have a moving average view that replaces each point by the average of the last N points.

**Comment by liufengyun** [comment]
@smarter To understand you correctly, you mean to do this for the `time` mode, or for `commit` mode as well?  (note: mode switch is in sidebar)

**Comment by smarter** [comment]
This should be useful in any mode.

**Comment by liufengyun** [comment]
So I understand you mean adding a new line, instead of hiding the authentic line (hiding is contentious).

**Comment by smarter** [comment]
Yes. We could also have some checkboxes to toggle which lines should be displayed or not.
